### ISSUE TITLE
full-stop: I can put a stop in where there already is one

### ISSUE THREAD

**Issue by rabsef-bicrym** [dalten-collective/full-stop#4]
![image](https://user-images.githubusercontent.com/52549148/215292865-f2fa09da-df3a-4e36-a928-cc144368a1ba.png)
The period starting on the 21st was closed on the 28th prior to entering the closure on the 26th. The value of the 26th sticks, losing the 28th.

**Comment by rabsef-bicrym** [comment]
Let's just preclude this behavior for now - we need a better way of identifying what pokes you'd need to issue to wipe out everything in a range and poke those down. that would be idea. For now, just make stopping a period permanent and not replaceable in this way.

**Comment by sink666** [comment]
Making permament, as in, once you've issued a stop on an open period the option in the menu is perhaps removed?

**Comment by rabsef-bicrym** [comment]
Yes... I think? Perhaps this is an open question to the community.

**Comment by sink666** [comment]
See #9 for an implementation of something like this. Once a stop or start is logged, the menu changes to remove the options. Specifically on highlighting a day with a stop or start you get the option to remove it. The Spot and Rate options are removed on days where you shouldn't be able to do them.
